decidim-meetings/app/views/decidim/meetings/shared/_year_calendar.html.erb

```diff
@@ -0,0 +1,63 @@
+<% add_decidim_page_title(t("decidim.meetings.directory.meetings.index.meetings")) %>
+
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_meetings", media: "all" %>
+<% end %>
+
+<% content_for :js_content do %>
+  <%= javascript_pack_tag "decidim_meetings" %>
+<% end %>
+
+<% year = local_assigns[:year] || Date.current.year %>
+<% content_for :aside do %>
+  <h1 class="h3 md:h1 decorator"><%= t("decidim.components.meetings.name") %></h1>
```

Comment:
Same thing here, shouldn't this be the component name?

Reply:
This page should be removed. It looks like it's not gonna be implemented